reframe/core/buildsystems.py

```diff
@@ -655,6 +655,49 @@ def emit_build_commands(self, environ):
         return prepare_cmd + [' '.join(configure_cmd), ' '.join(make_cmd)]
 
 
+class EasyBuild(BuildSystem):
+    easyconfigs = fields.TypedField(typ.List[str])
+    options = fields.TypedField(typ.List[str])
+    emit_package = fields.TypedField(bool)
+    package_opts = fields.TypedField(typ.Dict[str, str])
+
+    def __init__(self):
+        super().__init__()
+        self.easyconfigs = []
+        self.options = []
+        self.emit_package = False
+
+    def emit_build_commands(self, environ):
+        easyconfigs = ' '.join(self.easyconfigs)
+
+        if self.emit_package:
+            self.options.append('--package')
+            for key, val in self.package_opts.items():
+                self.options.append(f"--package-{key}={val}")
+
+        options = ' '.join(self.options)
+
+        cmd = [f"EASYBUILD_BUILDPATH={self._eb_sandbox}/build \\",
```

Comment:
You can use `rf` strings here so you don't have to escape the `\`

Reply:
Also use single quotes for strings.